spec/reek/configuration/excluded_paths_spec.rb

```diff
@@ -12,9 +12,7 @@
       let(:paths) { [smelly_source_dir, file_as_path] }
 
       it 'raises an error if one of the given paths is a file' do
-        Reek::CLI::Silencer.silently do
-          expect { exclusions.add(paths) }.to raise_error(Reek::Errors::ConfigFileError)
-        end
```

Comment:
Interesting. I can't even recall why we had that there.

Reply:
The method emitted a warning and called `exit` at some point.